api/tests/dotnet/IndigoTest.cs

```diff
@@ -17,6 +17,18 @@ public void TestIndigoVersion()
             indigo.Dispose();
         }
 
+        [TestMethod]
+        public void TestIndigoGetOneBitsList()
+        {
+            var indigo = new Indigo();
+            var indigoObject = indigo.loadMolecule("C1=CC=CC=C1");
+
+            Assert.AreEqual(
+                "1698 1719 1749 1806 1909 1914 1971 2056",
+                indigoObject.fingerprint().oneBitsList(),
+                "same one bits as in string 1698 1719 1749 1806 1909 1914 1971 205");
```

Comment:
Why missed the last `6` in the string? (`205` instead `2056`)
Here and in the referred Java pull request (apparently copy-pasted).

Reply:
Yes, my bad.
Imho this data shouldn't be mentioned in the message parameter.